package.json

```diff
@@ -34,17 +34,24 @@
   },
   "license": "(BSD-3-Clause AND CC-BY-SA-4.0)",
   "dependencies": {
+    "rx": "4.0.8",
```

Comment:
Rx is on here twice. Are you manually editing this file? You should use npm to do this.